app/celery/letters_pdf_tasks.py

```diff
@@ -228,16 +229,27 @@ def send_letters_volume_email_to_dvla(letters_volumes, date):
         send_notification_to_queue(saved_notification, queue=QueueNames.NOTIFY)
 
 
-def send_dvla_letters_via_api(print_run_deadline_local):
+def send_dvla_letters_via_api(
+    print_run_deadline_local,
+    batch_size=32,  # can this be larger because each task only takes 1 arg?
```

Comment:
some quick testing, i could send a task invocation with 180 args - 182 failed

100 could be reasonable. worth a comment saying that this is tied to the number and size of arguments to ensure we go back to this in case we add other args in future

Reply:
Also worth considering if it’s good/bad/irrelevant for the batch size to be a multiple of the query limit here:
https://github.com/alphagov/notifications-api/blob/46ac04b8547a3a95941d84a7d87908caca8d6027/app/dao/notifications_dao.py#L789

Reply:
my head leans towards irrelevant - if we go to the database in-between sending shatter tasks or half-way through doesn't really matter in my head cos the extra data we'll hold in memory (100 UUIDs) is so piddly